app/models/application_experience.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationExperience < ApplicationRecord
```

Comment:
Is it worth putting validations here to mirror the `not null` constraints in the schema? (here or in a separate PR).

Also validations for `start_date` being earlier than `end_date`.

Reply:
I've added the presence validators and will update in a separate PR with `start_date_before_end_date`.